variables.tf

```
@@ -22,6 +22,11 @@ variable "enable_logging" {
   description = "Trigger to enable ALB logging"
 }
 
+variable "default_load_balancer_is_internal" {
+  description = "Boolean determining if the load balancer is internal or externally facing."
+  default     = "false"
```

why default value is false? as far as I see the most common usage of ALB is internal
```suggestion
  default     = "true"
```

---

Value was taken from original module